.github/workflows/rust.yml

```diff
@@ -113,6 +113,25 @@ jobs:
       - name: clippy wasm32
         run: ./scripts/clippy_wasm.sh
 
+  # requires a different toolchain from the other checks (nightly)
+  check_wasm_atomics:
+    name: Check wasm32+atomics
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - run: sudo apt-get update && sudo apt-get install libgtk-3-dev libatk1.0-dev
+
+      - name: Set up cargo cache
+        uses: Swatinem/rust-cache@v2
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: nightly
+          targets: wasm32-unknown-unknown
+          components: rust-src
+
+      - name: Check wasm32+atomics eframe with wgpu
+        run: RUSTFLAGS='-C target-feature=+atomics' cargo +nightly check -p eframe --lib --no-default-features --features wgpu --target wasm32-unknown-unknown -Z build-std=std,panic_abort
```

Comment:
@Wumpf we could also fix that to a specified nightly version, to avoid random breakage on a nightly update. Unsure if I should escalate that question to emilk